product_operating_unit/models/product_template.py

```diff
@@ -0,0 +1,35 @@
+##############################################################################
+#
+#    Copyright (c) 2019 brain-tec AG (http://www.braintec-group.com)
+#    License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+#
+##############################################################################
+
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+
+
+class ProductTemplate(models.Model):
+    _inherit = 'product.template'
+
+    def _default_operating_unit_ids(self):
+        return [(6, 0,
+                 [self.env['res.users'].operating_unit_default_get(
+                     self.env.uid).id])]
+
+    operating_unit_ids = fields.Many2many(
+        'operating.unit', 'product_operating_unit_rel',
+        string='Operating Unit',
+        default=_default_operating_unit_ids
+    )
```

Comment:
@BT-rmartin What do you think of defining the OU on the product category and have a related field on product template?